thangs_importer.py

```diff
@@ -297,12 +297,24 @@ def import_model(self):
             else:
                 print('STL Import')
                 bpy.ops.import_mesh.stl(filepath=self.file_path)
-        except:
+        except Exception as e:
             print('Failed to Import')
             self.failed = True
             self.importing = False
+            self.amplitude.send_amplitude_event("Thangs Blender Addon - import model", event_properties={
```

Comment:
Is this just something that carried over from a different PR?

Reply:
https://github.com/physna/thangs-blender-addon/pull/47/commits/88e26daf0453db2139eff9b88d4be135a2ff74ab